src/web/pages/scanners/dialog.js

```diff
@@ -125,10 +119,7 @@ const ScannerDialog = ({
   const handleTypeChange = (value, name) => {
     if (onScannerTypeChange) {
       value = parseInt(value);
-      const scan_credentials = filter(
-        credentials,
-        client_cert_credentials_filter,
-      );
+      const scan_credentials = filter(credentials);
 
       onScannerTypeChange(value, name);
       onScannerTypeChange(
```

Comment:
Is this and the declaration of `scan_credentials` still needed or can it be removed?